src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  const price = days * cost;
+
+  if (days >= 7) {
+    return price - bigDiscount;
+  } else if (days >= 3) {
+    return price - smallDiscount;
+  }
```

Comment:
You could provide constants for those days values
```suggestion
  if (days >= LONG_TERM) {
    return price - bigDiscount;
  } else if (days >= SHORT_TERM) {
    return price - smallDiscount;
  }
```